app/models/spree/pay_easy.rb

```diff
@@ -0,0 +1,26 @@
+module Spree
+  class PayEasy < Spree::Base
+    belongs_to :payment_method
+    belongs_to :user, class_name: Spree.user_class, foreign_key: 'user_id'
+    has_many :payments, as: :source
+
+    validates :email, :given_name, :family_name, :given_name_kana, :family_name_kana, presence: true
```

Comment:
We should validate the numericality of `phone`, if it is not numerical komoju crashes and burns with a 500 server error.